cmd/kots/cli/check-version.go

```diff
@@ -0,0 +1,98 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/pkg/buildversion"
+	"github.com/replicatedhq/kots/pkg/handlers"
+	"github.com/replicatedhq/kots/pkg/k8sutil"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/spf13/viper"
+)
+
+// Checks the KOTS CLI version against the API version
+func cliVersionCheck() error {
+	cliVersion := buildversion.Version()
+
+	v := viper.GetViper()
+
+	log := logger.NewCLILogger()
+
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+
+	clientset, err := k8sutil.GetClientset()
+	if err != nil {
+		return errors.Wrap(err, "failed to get clientset")
+	}
+
+	namespace := v.GetString("namespace")
+	if err := validateNamespace(namespace); err != nil {
+		return errors.Wrap(err, "failed to validate namespace")
+	}
+
+	getPodName := func() (string, error) {
+		return k8sutil.FindKotsadm(clientset, namespace)
+	}
+
+	localPort, errChan, err := k8sutil.PortForward(0, 3000, namespace, getPodName, false, stopCh, log)
+	if err != nil {
+		return errors.Wrap(err, "failed to start port forwarding")
+	}
+
+	go func() {
+		select {
+		case err := <-errChan:
+			if err != nil {
+				log.Error(err)
+			}
+		case <-stopCh:
+		}
+	}()
+
+	getMetadataURL := fmt.Sprintf("http://localhost:%d/api/v1/metadata", localPort)
```

Comment:
Can we change this to hit /healthz instead?   Metadata specifically refers to application metadata.  And there's already a version field that needs a real value: https://github.com/replicatedhq/kots/blob/10b6d379ae2b7ade0e482e3ebbb9fe22709da00e/pkg/handlers/healthz.go#L33

Reply:
Perfect.  Will make this edit...